### ISSUE TITLE
Review `topic/drawable` PRs

### ISSUE THREAD

**Issue by louwers** [maplibre/maplibre-native#1389]
Tracking issue for the PRs of the [Renderer Modularization](https://github.com/maplibre/maplibre-native/blob/main/design-proposals/2022-10-27-rendering-modularization.md) effort. Reviews, comments & questions are very welcome.

## PRs

- https://github.com/maplibre/maplibre-native/pull/1355
- https://github.com/maplibre/maplibre-native/pull/1459
- https://github.com/maplibre/maplibre-native/pull/1501

## Process

`topic/drawable` will be merged into `topic/drawable-reviewed`. This will take several PRs. Once this is done `topic/drawable-reviewed` will be merged into `main`. For that last step we will need to ensure CI passes for the legacy renderer and that runs for the drawable renderer can be requested.

**Comment by louwers** [comment]
Some recent contributors that might want to have a look at these PRs.

@thehoneymad @tdcosta100 @fynngodau @birkskyum @ntadej @SiarheiFedartsou @boldtrn @jwinarske @etnav @roblabs @acalcutt @Vogtinator @artakka

**Comment by louwers** [comment]
Second PR is up: https://github.com/maplibre/maplibre-native/pull/1459

**Comment by louwers** [comment]
We had a Q&A about the [third PR](https://github.com/maplibre/maplibre-native/pull/1501) yesterday. I made a summary to the best of my abilities: https://github.com/maplibre/maplibre-native/discussions/1532

**Comment by louwers** [comment]
The focus is now on merging `topic/drawable` into `main`.

While all the code has been reviewed at this point, final questions & comments can be added there.

https://github.com/maplibre/maplibre-native/pull/1026

**Comment by louwers** [comment]
Closing this because #1570 has been merged.
